src/components/stepforms/ReplaceStepForm.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div>
+    <step-form-title :title="title"></step-form-title>
+    <ColumnPicker
+      id="searchColumnInput"
+      v-model="editedStep.search_column"
+      name="Search in column..."
+      placeholder="Enter a column"
+    ></ColumnPicker>
+    <WidgetInputText
+      id="newColumnInput"
+      v-model="editedStep.new_column"
+      name="(Optional) Write output in..."
+      placeholder="Enter a new column name"
+    ></WidgetInputText>
+    <WidgetList
+      addFieldName="Add a value to replace"
+      id="toReplace"
+      name="Values to replace:"
+      v-model="toReplace"
+      :defaultItem="[]"
+      :widget="widgetToReplace"
+      :automatic-new-field="false"
+    ></WidgetList>
+    <step-form-buttonbar :errors="errors" :cancel="cancelEdition" :submit="submit"></step-form-buttonbar>
+  </div>
+</template>
+
+<script lang="ts">
+import { Prop } from 'vue-property-decorator';
+import { Getter } from 'vuex-class';
+import { StepFormComponent } from '@/components/formlib';
+import ColumnPicker from '@/components/stepforms/ColumnPicker.vue';
+import WidgetInputText from '@/components/stepforms/WidgetInputText.vue';
+import WidgetList from '@/components/stepforms/WidgetList.vue';
+import WidgetToReplace from '@/components/stepforms/WidgetToReplace.vue';
+import BaseStepForm from './StepForm.vue';
+import { ReplaceStep } from '@/lib/steps';
+import { DataSetColumn } from '@/lib/dataset';
+
+@StepFormComponent({
+  vqbstep: 'replace',
+  name: 'replace-step-form',
+  components: {
+    ColumnPicker,
+    WidgetInputText,
+    WidgetList,
+  },
+})
+export default class ReplaceStepForm extends BaseStepForm<ReplaceStep> {
+  @Prop({ type: Object, default: () => ({ name: 'replace', search_column: '', to_replace: [[]] }) })
+  initialStepValue!: ReplaceStep;
+
+  @Getter columnHeaders!: DataSetColumn[];
+
+  readonly title: string = 'Replace values';
+  widgetToReplace = WidgetToReplace;
```

Comment:
`widgetToReplace` seems like odd name. I know that we've taken the bad habit to prefix (instead of suffix) widget class names with `widget` but it feels awkward here :/

Reply:
Hum ok but it mirrors the aggregation and filter steps. Every component that we use to feed the WidgetList component is prefixed with `widget`...
If we decide it is not a good naming, I think we should rather open a specific PR to clean the naming everywhere to keep consistency.

Reply:
yes, I did not mean to change it now. It will go in a specific PR